site/gatsby-site/cypress/plugins/index.js

```diff
@@ -21,6 +21,7 @@ module.exports = (on, config) => {
   config.env.e2eUsername = process.env.E2E_ADMIN_USERNAME;
   config.env.e2ePassword = process.env.E2E_ADMIN_PASSWORD;
   config.env.realmAppId = process.env.GATSBY_REALM_APP_ID;
+  config.env.isEmptyEnvironment = process.env.IS_EMPTY_ENVIRONMENT == 'true';
```

Comment:
just double-checking -- this should be a string compare? the ENV var isn't read as a boolean, or?

Reply:
Yes, this should be a string compare because the process.env variable is a string. I prefer to do this comparison here and use the `isEmptyEnvironment` setting as a boolean in the rest of the code